spec/unit/lib/temporal/activity/task_processor_spec.rb

```diff
@@ -53,6 +53,8 @@
         allow(client)
           .to receive(:respond_activity_task_failed)
           .and_raise(StandardError)
+        # Skip sleeps during retries to speed up the test.
+        expect(Temporal::Client::Retryer).to receive(:sleep).at_least(1).times
```

Comment:
nit: we're trying not to use `expect().to receive()` construct in specs because it conflates two thing — stubbing and checking. Instead all the spec explicitly stub via `allow().to receive()` and then check with `expect().to have_received()`